### ISSUE TITLE
Stuck in Fastboot3DS menu, cannot boot to regular 3DS Screen

### ISSUE THREAD

**Issue by stampylongr** [derrekr/fastboot3DS#33]
*(Originally posted at [GBATemp](https://gbatemp.net/threads/stuck-in-fastboot3ds-menu-cannot-boot-to-regular-3ds-screen.530769/))*

OK, so my original SD card got corrupted, and I didn't want to go through and replace the CFW and homebrew software and I found this: 3ds CFW: Restoring functionality on a new SD card. Put it all on my SD card, and all worked fine.

After a few days, I wanted to update B9S. So I, being the naive person that I am, decided to launch OPENFIRMINSTALLER instead of SafeB9SInstaller. They looked the same to me, and at the time I didn't know Fastboot3DS was on the SD card. I installed it, and now I am stuck on the Fastboot3DS screen, with NO WAY OF LAUNCHING LUMA3DS! Not to mention that the SD card will not mount, and every time I try to get into the NAND, it freezes up completely.

So, yeah. I'm stuck.

TL;DR: I need help.

**Comment by d0k3** [comment]
Seriously? You "accidentially" installed fastboot3ds, while trying to update a software that hasn't had an update in one and a half years? And you didn't even bother looking around the menu for a way to launch a FIRM file? Or, maybe, as you already found your way here, taking a very quick look into the readme that's *right there on the frontpage*?

Don't want to be offsensive, but... Oh well...

If your SD card does not mount in fb3ds, there is a problem with it. All homebrew FIRMs (that includes b9s, Luma 3DS, GM9, etc...) share the same basic SD card driver. If it doesn't work in one, it doesn't work in another. Check that SD card in a PC.

What do you mean by NAND freezing? Can you describe this in a little more detail? What are you trying to do and what exactly happens?

After you checked your SD card, maybe fixed the corruption, follow the quick start guide right there in the readme. Or, you know, just launch SB9SI / OFI again to switch back to b9s.
